szperacz/logger.py

```diff
@@ -1,18 +1,36 @@
 
 
 class Logger:
+    global log_level
+    log_level = 0
+    level = log_level
     DEBUG = '[DEBUG]'
     INFO = '[INFO]'
     ERROR = '[ERROR]'
 
+    error = open("error_log.txt", mode='w')
+
+    def set_log_level(self):
+        pass
```

Comment:
Body of this function is missing.

It should take an argument and set the `log_level` to given value.

Ideally, it should throw an exception once the given level is outside of `0 <= x <= 2` range.